Kiley dev

kileyhartigan opened the pull request:
In this pull request are changes to the 'now-to-update-these-docs' document to include GitHub desktop instructions and to propose a troubleshooting documentation framework/workflow. I then created an example troubleshooting document (about solving a simple mkdocs error) that aligns with the proposed troubleshooting framework.

kileyhartigan committed:
Troubleshooting workflow

1. Added to the 'how-to-update-these-docs' to include github desktop instructions and a workflow to contribute to the troubleshooting folder.

kileyhartigan committed:
Troubleshooting example doc

Created an example trouble shooting document that follows the trouble shooting framework laid out in the 'how-to-update-to-these-docs' doc.

kileyhartigan committed:
Navigation Changes

Trying to figure out how to get the navigation tabs to look the way I want.

kileyhartigan committed:
Lit review workflow

Added a possible lit review workflow to how-to-update-these-docs.md

Commit:
changes from jon

jonmatthis commented:
Hey @kileyhartigan  - I did a quick video run through with a bunch of unstructured thoughts about this PR: https://youtu.be/XQh7KumcCdw

Easier to just ramble at a camera than try to type it all out :P 

(I *just* uploaded it, so you might need to wait a sec before its ready to view)

kileyhartigan committed:
Literature organization considerations

How do we want to include publications in this knowledge base? There's an easy way and harder way.

kileyhartigan committed:
Spinal Level Reflex section outline

Beginning to build out spinal level reflexes resource library

kileyhartigan committed:
Adjusting troubleshooting and literature review workflows

Updating current workflows.

kileyhartigan committed:
Publication pdf thoughts

Adding snippets...

kileyhartigan committed:
Adding spinal level reflex library

Adding outlines for spinal level reflex papers.

jonmatthis commented on mkdocs.yml:
```diff
@@ -3,15 +3,27 @@ site_url: https://HuMoN-Research-Lab.github.io
-    - Home: index.md
-    - How to update these docs: how-to-update-these-docs.md
+plugins:
+  - glightbox
```
what does `glightbox` do?

kileyhartigan replied:
Copy that Jon, I'll make that adjustment.

jonmatthis reviewed (changes requested):
This is looking nice!

My main comment before you can merge is that I don't think "Spinal Level Reflexes" should be a 'top level' category. Could you make another category called something like "Literature Review" or something and move the "spinal level reflexes" directory into that?
